ansible/packages.yml

```diff
@@ -1,3 +1,5 @@
+---
```

Comment:
Why are these necessary?

Reply:
I understand the multiple files thing. But... why would you like to separate multiple documents if there's just one? I mean, I've seen this a couple times, but still don't get it why is this supposed to be a good practice. Extra lines which help... nothing? No push against, I'm just wondering... and trying to understand.